Harvest.Core/Domain/Ticket.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Harvest.Core.Domain
+{
+    public class Ticket
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public int ProjectId { get; set; }
+        public Project Project { get; set; }
+        public int? InvoiceId { get; set; }
+        public Invoice Invoice { get; set; }
+
+        [Required]
+        public int CreatedById { get; set; }
+        public DateTime CreatedOn { get; set; }
+
+        public int UpdatedById { get; set; }
+        public DateTime UpdatedOn { get; set; }
+
+        [StringLength(50)]
+        [Display(Name = "Task Name")]
+        public string Name { get; set; }
+        [Required]
+        public string Requirements { get; set; }
+        public DateTime DueDate { get; set; }
+
+        public string WorkNotes { get; set; } //For the person doing the Task

```

Comment:
Probably want to make this a Max length string

Reply:
It defaults to max. Doesn't seem to be a simple way of explicitly doing it.

Reply:
i would think you could just do [MaxLength(int.Max)]` or in the prop.  not sure if there's a better way